kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kms/aws/AwsKms.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kms.aws;
+
+import java.net.URI;
+
+import io.kroxylicious.systemtests.executor.Exec;
+import io.kroxylicious.systemtests.executor.ExecResult;
+
+/**
+ * The type Aws kms local.
+ */
+public class AwsKms implements AwsKmsClient {
+    private static final String AWS_CMD = "aws";
```

Comment:
Should the DEV_GUIDE call out that AWS cli is required?

Reply:
Yeah, only if an AWS cloud account is used. I will indicate that